packages/flutter_blue_plus_platform_interface/lib/src/bluetooth_msgs.dart

```diff
@@ -93,6 +94,7 @@ class BmScanSettings {
     required this.androidLegacy,
     required this.androidScanMode,
     required this.androidUsesFineLocation,
+    required this.webOptionalServices,
```

Comment:
I would suggest making this optional (or default to an empty list) to cover other non-web platforms.

```suggestion
    this.webOptionalServices,
```

Reply:
i disagree. just use an empty list

Reply:
> just use an empty list

```suggestion
this.webOptionalServices = const [],
```

> maybe I don't fully understand your motivation

I am simply attempting to reduce the amount of maintenance work required in each federated platform implementation as that is something that you originally raised concerns about.

Reply:
got it. so far its been pretty low key. kinda quiet even.